Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_boss_floor_2.prefab

```diff
@@ -11,6 +11,7 @@ GameObject:
   - component: {fileID: 3325143161705299546}
```

Comment:
I wasn't able to see the click feedback on top of the floor near the chest (around stage left area). Might need to play with heights here.

Reply:
Well that's quite mysterious. It's flat, so I don't know why it worked for only one half of the dais--and click feedback was already basing itself off the y height of the object that was clicked on. I have nudged up the click feedback hover height to to 0.15 (from 0.1), and now it appears everywhere.